LivrETS/Scripts/managefairs-calendarhandling.js

```diff
@@ -72,31 +72,31 @@ $(document).ready(function () {
                 break;
 
             case END_DATE_ID:
-                endDate = date.date.startOf("day");
+                endDate = date.date.startOf("day").add(1, 'days').subtract(1,'seconds');
```

Comment:
J'amènerais ton attention sur cette méthode de la lib momentjs: http://momentjs.com/docs/#/manipulating/end-of/

svp, corriger.